Tools/scripts/stable_abi.py

```diff
@@ -259,6 +259,37 @@ def gen_doc_annotations(manifest, args, outfile):
             'added': item.added,
             'ifdef_note': ifdef_note})
 
+@generator("ctypes_test", 'Lib/test/test_stable_abi_ctypes.py')
+def gen_ctypes_test(manifest, args, outfile):
+    """Generate/check the ctypes-based test for exported symbols"""
+    write = partial(print, file=outfile)
+    write(textwrap.dedent('''
+        # Generated by Tools/scripts/stable_abi.py
```

Comment:
Should this be checked by the _"Check if generated files are up to date"_ CI?

Reply:
Yes. I made the classic mistake, renaming without running the tests :/